submissions/phpslonuk/Popup/index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hooli-style Popup</title>
+  <link rel="shortcut icon" type="image/png"
+    href="https://raw.githubusercontent.com/phpslonuk/Hooli-style_Popup/main/popup-icons/favicon.png" />
```

Comment:
A little fix (I missed it) - the single tags should be without slash "/"(according to HTML5 specification)

Reply:
well, I will consider in the future